### ISSUE TITLE
Expand conflicts with Emmet

### ISSUE THREAD

**Issue by tobalsan** [Anomareh/PHP-Twig.tmbundle#43]
Hi,

since last update where you replaced `blockb` with `block.`, `ifb` with `if.` for twig multiline blocks, if one has Emmet extension installed, there is a conflict between both plugins in text expansion, and Emmet takes priority, which makes `block.` expand to `<block class=""></block>`.

**Comment by alimranahmed** [comment]
Expecting instructions or usages section in the readme file. When I install time this package, I was confused about how to use this package actually.  I think a readme file will make this package more sensible and more easy to use. Thank you.

**Comment by or3lie** [comment]
Hi @Anomareh ! What do you think about the colon solution ?

**Comment by diegocastro** [comment]
+1 for the colon.

**Comment by tobalsan** [comment]
yeah, it would be really cool, but looks like @Anomareh is very busy lately :P

**Comment by diegocastro** [comment]
@Anomareh if you're busy I could make the changes. It's something that is annoying me to no end.
